File: Commonrecycler-kotlin/src/main/java/com/shuyu/commonrecycler/xrecycler/other/ItemTouchHelperAdapter.java
package com.shuyu.commonrecycler.xrecycler.other;


/**
 * Created by jianghejie on 16/6/20.
 */

public interface ItemTouchHelperAdapter {

    void onItemMove(int fromPosition, int toPosition);


    void onItemDismiss(int position);
}

File: CommonRecycler-Kotlin/src/main/java/com/shuyu/commonrecycler/xrecycler/other/ItemTouchHelperAdapter.kt
package com.shuyu.commonrecycler.xrecycler.other


/**
 * Created by jianghejie on 16/6/20.
 */

open interface ItemTouchHelperAdapter {

    fun onItemMove(fromPosition: Int, toPosition: Int)


    fun onItemDismiss(position: Int)
}
